### PR TITLE
Change TxBuilder to look up keys from the config file

### LINKED ISSUE #82: Use the configured keypair for `TxBuilder`
Currently, we read seeds from the config file, but do not use them.
Since the `TxBuilder` supports it, we should start using them, but keep a backward-compatibility option to use the WK ones (especially for the integration tests of Agora).

### PR DESCRIPTION
**Pull request by Geod24**
This way we can deploy a TestNet where the keys are not known to everyone.

Fixes #82

### CODE REVIEW

**Review by kchulj — APPROVED** [review]
✅

**Comment by kchulj** [comment]
Needs rebase, currently testing locally.

**Comment by kchulj** [comment]
🟢 
Tested locally and works.

**Comment by Geod24** [comment]
We also need to add it when `None` is used no ?

**Comment by kchulj** [comment]
> We also need to add it when `None` is used no ?

Yes good point.

File: source/faucet/config.yaml [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 # When we deploy Faucet on the real TestNet, the keys used will not be well-known.
 keys:
+  # Genesis seed
+  - SDN7BBGE6Z6OQM3K4PACLTZUJ5QX4AY4QPDQ2JJ2JCFWCG2OIYYALIRY
```
Comment (hewison-chris) [inline source/faucet/config.yaml:8]: Perhaps the comment can go at end of line to make it clear it is just that line.

Comment (hewison-chris) [reply source/faucet/config.yaml:8]: Although maybe it is not possible.

Comment (kchulj) [reply source/faucet/config.yaml:8]: Or another comment for the 6 keys below?

**Comment by Geod24** [comment]
Also need a few other fixes, hold on

**Comment by Geod24** [comment]
Updated with a few improvements:
1) We already had `getOwnedUTXO`, so I moved the filter call there;
2) If we need `Genesis` in the config file, we need it with `none`;
3) Moved the config file in its directory, like Agora, because code and example config should not mix;
4) Make the config file a little bit better w.r.t. real world usage by using A-Z.

**Review by hewison-chris — APPROVED** [review]
LGTM

**Review by kchulj — APPROVED** [review]

**Pull request merged**